projects/storefrontlib/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -23,6 +30,10 @@ import { PageTitleComponent } from '../page-header/page-title.component';
 })
 export class BreadcrumbComponent extends PageTitleComponent implements OnInit {
   crumbs$: Observable<any[]>;
+  ariaLive$: Observable<boolean> = of(true);
```

Comment:
```suggestion
  ariaLive$: Observable<boolean> = 
    this.featureConfigService.isEnabled('a11yRepeatedPageTitleFix') 
      ? this.router.events.pipe(
        filter((e) => e instanceof NavigationEnd),
        map(() => {
          return document.activeElement !== document.body;
        })
      )
    : of(true);
```

Reply:
Perhaps we could set the observable like this.

Reply:
Done ;)